### PR TITLE
clean up package.json

### PR DESCRIPTION
**Pull request by reggi**
* Added server to `npm test`, was just testing client before (very confusing)
* Any bin that is installed within the package scope can be called directly by it's command so no need to `./node_modules/package/bin/package`
* Add linter `npm run lint` and added it to `npm test`

just some ideas

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,13 +6,14 @@
   "scripts": {
     "start": "node server",
     "dev": "./scripts/dev.sh",
-    "test": "npm run test-client",
-    "test-client": "node ./node_modules/karma/bin/karma start test/karma-dev.conf.js --single-run --browsers PhantomJS",
-    "test-client-ci": "node ./node_modules/karma/bin/karma start test/karma-ci.conf.js --single-run --browsers PhantomJS",
-    "test-watch": "node ./node_modules/karma/bin/karma start test/karma-dev.conf.js --browsers PhantomJS",
+    "lint": "eslint ./",
```
Comment (daviskoh) [inline package.json]: do we need to ignore `/node_modules` if we do this?

Comment (reggi) [inline package.json]: > In addition to any patterns in a .eslintignore file, ESLint always ignores files in node_modules/**.

http://eslint.org/docs/user-guide/configuring.html

Comment (daviskoh) [inline package.json]: :+1: :zap:

Comment (jackofseattle) [inline package.json]: I use test-watch quite often when doing client development.  would be nice to leave that in.
Otherwise, LGTM

Comment (reggi) [inline package.json]: Whoops @jackofseattle didn't mean to delete test-watch :(

**Pull request closed without merging**